Bottom: eb5ef38fd05bea94d865310ae5a3ed1eecadc717
Top:    57af8232920d29df08d79813009ffc5785ed8c33
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 01:59:33 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 1431d93..827901d 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -389,6 +389,8 @@ ghid_set_color (hidGC gc, const char *name)
   double r, g, b, a;
   a = 1.0;
 
+  current_gc = gc;
+
   if (old_name != NULL)
     {
       if (strcmp (name, old_name) == 0)
